File: Sistema/src/app/models/CertificadoDadosLoja.Model.ts
export class CertificadoDadosLojaModel {
  lojCodi: number;
  lojNome: string;
  lojNumL: string;
  potSigl: string;
  potNome: string;
  sesDtHr: Date;
  cidNome: string;
  estSigl: string;
  lojLogo: string;
  potLogo: string;

  constructor() {
    this.lojCodi = 0;
    this.lojNome = '';
    this.lojNumL = '';
    this.potSigl = '';
    this.potNome = '';
    this.sesDtHr = new Date();
    this.cidNome = '';
    this.estSigl = '';
    this.lojLogo = '';
    this.potLogo = '';
  }
}

File: API/Models/CertificadoDadosLojaModel.cs
namespace API_Visitatus.Models
{
    public class CertificadoDadosLojaModel
    {
        public long LojCodi { get; set; }
        public string? LojNome { get; set; }
        public string? LojNumL { get; set; }
        public string? PotSigl { get; set; }
        public string? PotNome { get; set; }
        public DateTime SesDtHr { get; set; }
        public string? CidNome { get; set; }
        public string? EstSigl { get; set; }
        public string? LojLogo { get; set; }
        public string? PotLogo { get; set; }
    }
}
